src/api/app/models/event/base.rb

```diff
@@ -40,6 +41,13 @@ def payload_keys(*keys)
         @payload_keys += keys
       end
 
+      def shortenable_key(key = nil)
+        # this function serves both for reading and setting
```

Comment:
Yes I know we are not Java devs so shouldn't have to write getter/setter methods but thats how the Event classes are already implemented and although this is kind of annoying it still works. When there is time for it I would like to get rid of all these class methods in the Event classes though.

Reply:
Then add at least a `FIXME` please